.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+Please start by describing concisely the nature of the bug or the feature
+request.
+
+In case of a bug report, please describe the steps how the problem may be
+reproduced. Does the problem occur on production servers? Does it work on the
+development server? Can you reproduce it locally on the `master` branch?
```

Comment:
I would limit mention of branches here, again. Most of the issues come from us on the team, and we may not all be building the instance to test it.